apps/google-analytics-4/frontend/src/components/main-app/ChartContent.tsx

```diff
@@ -0,0 +1,31 @@
+import { Row, RunReportResponse } from '@/types';
+import LineChart from './line-chart/LineChart';
+
+interface Props {
+    pageViewData: RunReportResponse;
+}
+
+const ChartContent = (props: Props) => {
+    const { pageViewData } = props;
+
+    const parseRowViews = (): number[] => {
+        return pageViewData.rows.map((r: Row) => +r.metricValues[0].value)
```

Comment:
`+r.metricValues[0].value` I'm not familiar with this syntax, whats's the + doing?

Reply:
It's a unary operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus . Its purpose is to convert a non-number type to a number (in this case, a string value of `'17'`. Its considered a little more performant than using the `Number()` class since it doesn't create a new instance of the class but does the conversion natively.

Reply:
Ah cool, thanks!

Reply:
this is great to know!